README.md

```diff
@@ -223,40 +155,24 @@ Feel free to check the [issues page](../../issues/).
 
 <!-- SUPPORT -->
 
-## ⭐️ Show your support <a name="support"></a>
+## ⭐️ Show your support <a name="support" href = "https://github.com/JohnteFresh">GitHub</a>
 
-If you like this project leave a review and rate us on Google playstore and AppleStore
+If you like this project feel free to contribute and leave a review.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+## 🙏 Acknowledgments <a name="acknowledgements">Lewis Hinga</a>
 
 I would like to thank Lewis Hinga for his valued contribution and help in completion of the project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-- **[How much do you charger?]**
-
-  - [0 upfront fee.]
-
-- **[Can i use the app on my phone?]**
-
-  - [You can use a laptop, a phone, any smart device can be used to access.]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT.md] licensed.
```

Comment:
- [OPTIONAL] Please correct your license path because on README file your link to license is not visible. So please correct your path by looking at the below suggestion.

```suggestion
This project is [MIT](./MIT.md) licensed.
```